pandas-stubs/core/series.pyi

```diff
@@ -155,6 +161,16 @@ class Series(IndexOpsMixin, NDFrame, Generic[S1]):
         fastpath: bool = ...,
     ) -> TimestampSeries: ...
     @overload
+    def __new__(
+        cls,
+        data: PeriodIndex,
+        index: Optional[Union[_str, int, Series, List, Index]] = ...,
+        dtype=...,
+        name: Optional[Hashable] = ...,
```

Comment:
Could be shortened to `Hashable`. `None` is `Hashable`. Could also keep it to be more explicit.

Reply:
> Could be shortened to `Hashable`. `None` is `Hashable`. Could also keep it to be more explicit.

For consistency, I think I will keep it to be more explicit.  Not sure that people know that `None` is hashable.